services/src/pro/handlers/drone_mapping.rs

```diff
@@ -325,7 +325,7 @@ async fn unzip(zip_path: &Path, target_path: &Path) -> Result<(), error::Error>
     let zip_path = zip_path.to_owned();
     let target_path = target_path.to_owned();
 
-    tokio::task::spawn_blocking(move || {
+    geoengine_operators::util::spawn_blocking(move || {
```

Comment:
maybe reexport this as crate::util::spawn_blocking for easier readibility?

Reply:
f4a36a3